### DISCUSSION TITLE
余额查询失败

### DISCUSSION THREAD

**Discussion by Nemo00789** [ChatGPTNextWeb/NextChat#2536] [Q&A]
点击重新检查的时候会显示查询失败
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/133762219/2e4d93cf-4746-4f6d-abd2-d3637fb4c71d)

这个东西是有bug? 还是说我本地部署的内容有问题...
部署版本20230524, 没有使用docker

**Comment by hsl333** [comment]
{
  "error": {
    "message": "Your request to GET /dashboard/billing/usage must be made with a session key (that is, it can only be made from the browser). You made it with the following key type: secret.",
    "type": "server_error",
    "param": null,
    "code": null
  }
}我也是提示以上信息

**Comment by misakaok** [comment]
最新版本看不到余额
